README.md

```diff
@@ -614,8 +616,196 @@ let sameFont = UIFont(font: FontFamily.Helvetica.Bold, size: 16.0) // iOS
 let sameFont = NSFont(font: FontFamily.Helvetica.Bold, size: 16.0) // OS X
 ```
 
----
+## Models
+
+```sh
+swiftgen json /path/to/model/file -t model
+swiftgen templates cat model-base > /path/to/output/file
+```
```

Comment:
I'm not really a fan of having a command that's effectively split into 2 dependent parts. I understand you want to reduce code duplication, and I wholeheartedly agree with that sentiment.

That said, would it be better to move the contents of `model-base` to a gist or mini-library? There are new swift JSON parsing libraries appearing every minute, and everyone has their preferences about which one to use. I'm just wondering if it's worth adding a static template to SwiftGen just for this.